doc/fuzzing.md

```diff
@@ -150,16 +165,18 @@ $ git clone https://github.com/bitcoin/bitcoin
 $ cd bitcoin/
 $ git clone https://github.com/AFLplusplus/AFLplusplus
 $ make -C AFLplusplus/ source-only
-$ ./autogen.sh
 # If afl-clang-lto is not available, see
 # https://github.com/AFLplusplus/AFLplusplus#a-selecting-the-best-afl-compiler-for-instrumenting-the-target
-$ CC=$(pwd)/AFLplusplus/afl-clang-lto CXX=$(pwd)/AFLplusplus/afl-clang-lto++ ./configure --enable-fuzz
-$ make
-# For macOS you may need to ignore x86 compilation checks when running "make". If so,
-# try compiling using: AFL_NO_X86=1 make
+$ cmake -B build_fuzz \
+   -DCMAKE_C_COMPILER="$(pwd)/AFLplusplus/afl-clang-lto" \
+   -DCMAKE_CXX_COMPILER="$(pwd)/AFLplusplus/afl-clang-lto++" \
+   -DBUILD_FOR_FUZZING=ON
+$ cmake --build build_fuzz -j$(nproc)
+# For macOS you may need to ignore x86 compilation checks when running "cmake --build". If so,
+# try compiling using: `AFL_NO_X86=1 cmake --build build_fuzz -j$(nproc)`
```

Comment:
if this comment still refers to the mac build, note that mac doesn't have `nproc`

Reply:
cc @dergoegge

Reply:
Right, the "-j$(nproc)" bit should be removed. I don't have a Mac so I don't no if these instructions are still relevant.

Reply:
The mac equivalent is `make -j$(sysctl -n hw.ncpu)`

Reply:
Addressed in https://github.com/hebasto/bitcoin/pull/316.

Reply:
Resolved in https://github.com/bitcoin/bitcoin/pull/30454#issuecomment-2281111100.